### PR TITLE
Created CollectBall

### PR DESCRIPTION
**Pull request by KaneBarnes**
Created CollectBall command and minor changes to Intake subsystem

### CODE REVIEW

**Review by AngelaKimmy — COMMENTED** [review]
Please map command to a button

File: src/main/java/frc/robot/commands/CollectBall.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  @Override
+  public void initialize() {
+    // Start spinning intake motor
+    intake.setIntakeSpeed(1.0);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    intake.setIntakeSpeed(1.0);
```
Comment (AngelaKimmy) [inline src/main/java/frc/robot/commands/CollectBall.java:30]: We don't need to set intake speed here

**Review by TaylerUva — CHANGES_REQUESTED** [review]
You have crated the command but what makes it run? You must program it to run somehow.

**Review by TaylerUva — APPROVED** [review]

**Comment by TaylerUva** [comment]
closed #25

**Pull request merged**